Pandocker/21.02/MarkdownToPdf.nf

```diff
@@ -0,0 +1,24 @@
+process MarkdownToPdf {
+    tag {"Pandocker MarkdownToPdf"}
+    label 'Pandocker_21_02'
+    label 'Pandocker_21_02_MarkdownToPdf'
+
+    container = 'library://dalibo/pandocker:v21.02'
+    shell = ['/bin/bash', '-euo', 'pipefail']
+
+    input:        
+        path(md_file)
+     
+    output:        
+        file "output.pdf"
```

Comment:
- Use path instead of file. See: https://nextflow.io/docs/latest/process.html?highlight=path#output-path (_"The path qualifier should be preferred over file to handle process output files when using Nextflow 19.10.0 or later."_)
 - Use `()`
 - I don't like 'output.pdf' as an output file name, would probably require a lot of renaming? Maybe use `md_file.baseName`?

So to conclude, output would look like this:
```
 output:        
    path("${md_file.baseName}.pdf")
```

Reply:
Completely fair, changes implemented!